### PR TITLE
MPP

### PR DESCRIPTION
**Pull request by nulls**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -17,29 +15,39 @@ repositories {
...
+                implementation(kotlin("test"))
+                implementation("org.junit.jupiter:junit-jupiter-api:5.9.0")
+                runtimeOnly("org.junit.jupiter:junit-jupiter-engine:5.9.0")
+            }
         }
     }
+}
 
+tasks {
     withType<Javadoc>().configureEach {
```
Comment (nulls) [inline build.gradle.kts]: probably it should be rewritten to dokka

Comment (petertrr) [reply build.gradle.kts]: We have setup for publishing to Github packages in https://github.com/saveourtool/sarif4k/commits/feature/publishing, but Github doesn't require docs (unlike Maven Central)

Yes, dokka, artifact signing and publications should be set up, probably, similarly to how we are doing it in save-cli. However, folks at Detekt can have different conventions for these things, so probably we can simply switch Javadoc tasks with Dokka and suggest these changes as is

Comment (nulls) [reply build.gradle.kts]: added dokka, but I'm getting the following message in gradle:
```
> Task :publishSarif4kPublicationToMavenLocal
Multiple publications with coordinates 'io.github.detekt.sarif4k:sarif4k:0.3.0-SNAPSHOT' are published to repository 'mavenLocal'. The publications 'kotlinMultiplatform' in root project 'sarif4k' and 'sarif4k' in root project 'sarif4k' will overwrite each other!
```

And it produces the following artifacts:
```
$ ls ~/.m2/repository/io/github/detekt/sarif4k/*/0.3.0-SNAPSHOT
/c/Users/a00665046/.m2/repository/io/github/detekt/sarif4k/sarif4k/0.3.0-SNAPSHOT:
maven-metadata-local.xml    sarif4k-0.3.0-SNAPSHOT.module  sarif4k-0.3.0-SNAPSHOT-javadoc.jar                   sarif4k-0.3.0-SNAPSHOT-sources.jar
sarif4k-0.3.0-SNAPSHOT.jar  sarif4k-0.3.0-SNAPSHOT.pom     sarif4k-0.3.0-SNAPSHOT-kotlin-tooling-metadata.json

/c/Users/a00665046/.m2/repository/io/github/detekt/sarif4k/sarif4k-jvm/0.3.0-SNAPSHOT:
maven-metadata-local.xml  sarif4k-jvm-0.3.0-SNAPSHOT.jar  sarif4k-jvm-0.3.0-SNAPSHOT.module  sarif4k-jvm-0.3.0-SNAPSHOT.pom  sarif4k-jvm-0.3.0-SNAPSHOT-sources.jar

/c/Users/a00665046/.m2/repository/io/github/detekt/sarif4k/sarif4k-linuxx64/0.3.0-SNAPSHOT:
maven-metadata-local.xml  sarif4k-linuxx64-0.3.0-SNAPSHOT.klib  sarif4k-linuxx64-0.3.0-SNAPSHOT.module  sarif4k-linuxx64-0.3.0-SNAPSHOT.pom  sarif4k-linuxx64-0.3.0-SNAPSHOT-sources.jar

/c/Users/a00665046/.m2/repository/io/github/detekt/sarif4k/sarif4k-mingwx64/0.3.0-SNAPSHOT:
maven-metadata-local.xml  sarif4k-mingwx64-0.3.0-SNAPSHOT.klib  sarif4k-mingwx64-0.3.0-SNAPSHOT.module  sarif4k-mingwx64-0.3.0-SNAPSHOT.pom  sarif4k-mingwx64-0.3.0-SNAPSHOT-sources.jar
```

Comment (nulls) [reply build.gradle.kts]: resolved, removed setting artifactId explicitly

**Pull request closed without merging**